README.md

```diff
@@ -21,11 +21,22 @@ The data gathered here are compatible with [INS Data Model v1.0.1](https://githu
 
 The INS-Data repository workflow follows the general outline below:  
```

Comment:
I'm imagining this section as less of a table of contents and more of a listing of just the modules/steps. Can you change my old wording here to "The INS Data Gathering workflow consists of the following steps:" for clarity and then revert the additions in this section? 
Then adding an "Errors and Warnings" link to the Table of Contents should do it.

Reply:
Sure thing. Taking a look now.

Reply:
Looks like the changes to the table of contents were done accidentally by one of my VS Code extensions for Markdown. Sorry I didn't catch it earlier. I'll revert those changes and also make the wording change that you requested.

Reply:
Ah, that makes sense. All else looks good. Thanks!

Reply:
Great. And the changes have been made and are now ready.